strawberry_django/fields/field.py

```diff
@@ -0,0 +1,105 @@
+import strawberry
+from django.db import models
+from strawberry.arguments import UNSET, convert_arguments
+from strawberry.field import StrawberryField
+from .. import utils
+from ..resolvers import django_resolver
+from ..filters import StrawberryDjangoFieldFilters
+from ..ordering import StrawberryDjangoFieldOrdering
+from ..pagination import StrawberryDjangoPagination
+
+
+class StrawberryDjangoFieldBase:
+    def get_queryset(self, queryset, info, **kwargs):
+        type_ = self.type or self.child.type
+        get_queryset = getattr(type_, 'get_queryset', None)
+        if get_queryset:
+            queryset = get_queryset(self, queryset, info, **kwargs)
+        return queryset
+
+    @property
+    def django_model(self):
+        type_ = self.type or self.child.type
+        return utils.get_django_model(type_)
+
+
+class StrawberryDjangoField(
+        StrawberryDjangoFieldOrdering,
+        StrawberryDjangoFieldFilters,
+        StrawberryDjangoPagination,
+        StrawberryDjangoFieldBase,
+        StrawberryField):
+
+    def __init__(self, django_name=None, graphql_name=None, python_name=None, **kwargs):
+        self.django_name = django_name
+        self.is_auto = utils.is_auto(kwargs.get('type_', None))
+        self.is_relation = False
+        self.origin_django_type = None
+        self.input_type = None # only mutations have input type
+        super().__init__(graphql_name=graphql_name, python_name=python_name, **kwargs)
+
+    @classmethod
+    def from_field(cls, field, django_type):
+        if utils.is_strawberry_django_field(field) and not field.origin_django_type:
+            return field
+
+        default_value = getattr(field, 'default_value', getattr(field, 'default', UNSET))
+        new_field = StrawberryDjangoField(
+            base_resolver=getattr(field, 'base_resolver', None),
+            default_factory=field.default_factory,
+            default_value=default_value,
+            django_name=getattr(field, 'django_name', field.name),
+            graphql_name=getattr(field, 'graphql_name', None),
+            python_name=field.name,
+            type_=field.type,
+        )
+        new_field.is_auto = getattr(field, 'is_auto', False)
+        new_field.origin_django_type = getattr(field, 'origin_django_type', None)
+        return new_field
+
+    def get_result(self, source, info, kwargs):
```

Comment:
Shouldnt this be `**kwargs`? If not, find a different name.

Reply:
This is the name strawberry core uses. I wanted to use same naming here for consistency. I already proposed to use `* *kwargs` syntax for that.

See https://github.com/strawberry-graphql/strawberry/issues/920#issuecomment-842568355